src/presentational-components/shared/card-common.tsx

```diff
@@ -2,9 +2,13 @@
 import React, { Fragment } from 'react';
 import styled from 'styled-components';
 import { AnyObject } from '../../types/common-types';
+import Truncate from 'react-truncate';
+import { Tooltip } from '@patternfly/react-core';
 
 const CardPropText = styled.div`
   overflow: hidden;
+  text-overflow: ellipsis;
+  lines: 3;
```

Comment:
This CSS property works only in chrome and edge right now. I think `react-truncate` should work without it.

Reply:
@Hyperkid123 - you are correct, I meant to remove these lines.